### PR TITLE
[#9682] Rename the Dapps tab to Browser

### LINKED ISSUE #9682: Rename the Dapps tab to Browser
### Description

Please, let's rename the `Dapps` tab to `Browser` and change its icon. 

Screenshot 2019-12-17 at 09 39 11

You can find the browser icon [here](https://www.dropbox.com/sh/ca4tf7b6tqn8wba/AAC0oXqU0VALNf_HpvgImUh1a?dl=0) 

or in [Figma](https://www.figma.com/file/Gk13uZiVwGLHMPvQjaICAMWM/Status-icons?node-id=550%3A2)

### Acceptance Criteria

Icon and text label is changed

### Notes

I did the necessary red tape and it's approved 🙏

### PR DESCRIPTION
**Pull request by errorists**
fixes #9682 

### Summary

• Renamed the tab bar text label to 'Browser'
• Swapped the tab bar icon to browser
• Overwritten existing browser icon with a slightly refreshed one to better match the appearance of other icons in the tab bar

### Review notes
• baby's first PR, pls correct if necessary :) 

### Testing notes
• test if the change made matches this 

Screenshot 2019-12-17 at 09 39 11

#### Platforms

- Android
- iOS
#### Areas that maybe impacted
bottom Tab Bar

### Steps to test

- Open Status
- Verify the bottom tab bar's appearance

status: ready

### CODE REVIEW

File: translations/en.json [hunk 1]
```diff
@@ -278,7 +278,7 @@
 	"custom-networks": "Custom networks",
 	"dapp": "ÐApp",
 	"dapp-would-like-to-connect-wallet": "would like to connect to",
-	"dapps": "ÐApps",
```
Comment (flexsurfer) [inline translations/en.json]: this label is used in other places so its better to create new label browser and use it in tabs

File: translations/en.json [hunk 2]
```diff
@@ -278,7 +278,7 @@
...
+	"dapps": "Browser",
```
Comment (flexsurfer) [inline translations/en.json]:
```suggestion
        "dapps": "ÐApps",
	"browser": "Browser",
```

File: src/status_im/ui/components/tabbar/core.cljs [hunk 3]
```diff
@@ -38,7 +38,7 @@
     (when-not platform/desktop?
       {:nav-stack           :browser-stack
        :content             {:title (i18n/label :t/dapps)
```
Comment (flexsurfer) [inline src/status_im/ui/components/tabbar/core.cljs]:
```suggestion
       :content             {:title (i18n/label :t/browser)
```

**Review by flexsurfer — CHANGES_REQUESTED** [review]

File: translations/en.json (hunk 1 above)
Comment (errorists) [reply translations/en.json]: thanks, done!

**Review by flexsurfer — APPROVED** [review]

**Comment by Serhy** [comment]
Lgtm too.
Failed autotests i'm looking separately, they have nothing to do with this changes.
Thanks @errorists !

**Pull request merged**